packages/theme/components/layout/root-layout/Theme.tsx

```diff
@@ -130,6 +130,19 @@ export function Theme({pageMap, children}: ThemeProps) {
                 <Header flatDocsDirectories={flatDocsDirectories} siteTitle={siteTitle} pageMap={pageMap} />
               </PRCBox>
               <PageLayout rowGap="none" columnGap="none" padding="none" containerWidth="full">
+                <PageLayout.Pane
```

Comment:
Could you add a changeset please @danielguillan. Will just need to indicate that side navigation and content panes have switched document order for improved accessibility (or similar)